api/models.py

```diff
@@ -62,3 +62,17 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event(models.Model):
+
+	title = models.CharField(max_length=255, blank=False, null=False)
+	start_date = models.DateField(blank=False, null=False)
```

Comment:
Nope, it wasn't wrong, we have the same DateTimeField with Projects, Tasks. Change it back to the way it was.